### PR TITLE
[TR] PIM-4895: Fix redirect after mass edit unpublished products

### PR DESCRIPTION
**Pull request by pierallard**
| Q                  | A
| ------------------ | ---
| Migration script?  | n
| Behats             | y
| Specs & Static     | y
| Changelog updated  | n

### CODE REVIEW

File: src/Pim/Bundle/EnrichBundle/Controller/MassEditActionController.php [hunk 1]
```diff
@@ -200,6 +206,7 @@ public function performAction($operationAlias)
 
         $form = $this->massEditFormResolver->getConfigurationForm($operationAlias);
         $form->remove('operationAlias');
+
```
Comment (willy-ahva) [inline src/Pim/Bundle/EnrichBundle/Controller/MassEditActionController.php]: remove blank line

**Comment by willy-ahva** [comment]
GTM if CI ok and after little comments

**Comment by ClementGautier** [comment]
GTM

**Comment by solivier** [comment]
GTM

File: src/Pim/Bundle/EnrichBundle/Resources/config/controllers.yml [hunk 2]
```diff
@@ -185,6 +185,7 @@ services:
             - '@akeneo_batch.connectors'
             - '@pim_enrich.mass_edit_action.operation.registry'
             - '@pim_enrich.mass_edit_action.form_resolver'
+            - { family-grid: 'pim_enrich_family_index', default: 'pim_enrich_product_index' }
```
Comment (nidup) [inline src/Pim/Bundle/EnrichBundle/Resources/config/controllers.yml]: new argument = a BC break! please indicates it in the changelog

File: src/Pim/Bundle/EnrichBundle/Controller/MassEditActionController.php [hunk 3]
```diff
@@ -88,7 +92,8 @@ public function __construct(
         JobRepositoryInterface $jobRepository,
         ConnectorRegistry $connectorRegistry,
         OperationRegistryInterface $operationRegistry,
-        MassEditFormResolver $massEditFormResolver
+        MassEditFormResolver $massEditFormResolver,
+        array $gridNameRouteMapping
```
Comment (nidup) [inline src/Pim/Bundle/EnrichBundle/Controller/MassEditActionController.php]: to avoid the BC break we should add a default value to the argument

Comment (willy-ahva) [inline src/Pim/Bundle/EnrichBundle/Controller/MassEditActionController.php]: :+1:

Comment (pierallard) [inline src/Pim/Bundle/EnrichBundle/Controller/MassEditActionController.php]: @nidup My last commit is it what you want ?

**Pull request merged**